aten/src/ATen/Dispatch_v2.h

```diff
@@ -87,7 +87,7 @@
 
 #define AT_FLOAT8_TYPES                                          \
   c10::kFloat8_e5m2, c10::kFloat8_e5m2fnuz, c10::kFloat8_e4m3fn, \
-      c10::kFloat8_e4m3fnuz
+      c10::kFloat8_e4m3fnuz, c10::kFloat8_e8m0fnu
```

Comment:
Its not immediately clear why this should be in this macro

Reply:
why not?

Reply:
from offline discussion, I think @drisspg was asking if this is really a float8 type.  My answer is yes - this is a float8 type, although it's very special.  No sign bit (always positive), 8 exponent bits, mantissa bit is implied.

note that thinking about this as a true float8 type (and not something like "an exponent of a float8 type") matches LLVM (https://github.com/llvm/llvm-project/pull/111028) and jax ml-dtypes (https://github.com/jax-ml/ml_dtypes/pull/166)